Makefile

```diff
@@ -11,7 +11,7 @@ build: vendor
 	go build -o bin/torusblk -ldflags "-X $(REPOPATH).Version=$(VERSION)" ./cmd/torusblk
 
 test: bin/glide
-	go test $(shell ./bin/glide novendor)
```

Comment:
Go flags. I think it's just one dash. Plus this definitely runs the race detector